db/migrate/20200416212002_remove_rows_from_lab_record_imports.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveRowsFromLabRecordImports < ActiveRecord::Migration[5.2]
+  def up
+    remove_column :lab_record_imports, :rows
+  end
+
+  def down
+    add_column :lab_record_imports, :rows, :jsonb
+  end
+end
```

Comment:
You should configure your editor to always add a newline at EOF if missing 😉 
If it's vscode: `"files.insertFinalNewline": true`